лабораторная4/task1.py

```diff
@@ -0,0 +1,107 @@
+class Earphones:
+    """
+            Создание и подготовка к работе объекта "Наушники"
+            :param name: Марка, защищенный аргумент, пользователь не может создать новые виды наушников
+            :param manufacturer: Производитель, защищенный аргумент, пользователь не может создать новые виды наушников
+    """
+    def __init__(self, name: str, manufacturer: str):
+        self._name = name
+        self._manufacturer = manufacturer
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def manufacturer(self):
+        return self._manufacturer
+
+    @name.setter
+    def name(self, new_name: str):
+        self._name = new_name
+
+    @manufacturer.setter
+    def manufacturer(self, new_manufacturer: str):
+        self._manufacturer = new_manufacturer
+
+    def __str__(self):
+        return f'Наушники марки "{self._name}" производителя "{self._manufacturer}"'
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}: name={self._name!r}, manufacturer ={self._manufacturer})'

```

Comment:
```suggestion
        return f'{self.__class__.__name__}(name={self._name!r}, manufacturer ={self._manufacturer!r})'

```
Неверно задано использование f-строки